integration-tests/grpc/src/main/java/org/apache/camel/quarkus/component/grpc/it/GrpcRoute.java

```diff
@@ -16,22 +16,33 @@
  */
 package org.apache.camel.quarkus.component.grpc.it;
 
+import java.util.Optional;
+
 import org.apache.camel.Message;
 import org.apache.camel.builder.RouteBuilder;
 import org.apache.camel.quarkus.component.grpc.it.model.PingRequest;
 import org.apache.camel.quarkus.component.grpc.it.model.PongResponse;
+import org.eclipse.microprofile.config.Config;
+import org.eclipse.microprofile.config.ConfigProvider;
 
 public class GrpcRoute extends RouteBuilder {
 
     @Override
     public void configure() throws Exception {
-        from("grpc://localhost:9000/org.apache.camel.quarkus.component.grpc.it.model.PingPong?synchronous=true")
-                .process(exchange -> {
-                    final Message message = exchange.getMessage();
-                    final PingRequest request = message.getBody(PingRequest.class);
-                    final PongResponse response = PongResponse.newBuilder().setPongName(request.getPingName() + " PONG")
-                            .setPongId(request.getPingId()).build();
-                    message.setBody(response);
-                });
+        fromF("grpc://localhost:%d/org.apache.camel.quarkus.component.grpc.it.model.PingPong?synchronous=true",
```

Comment:
Do we need this? IIRC the test port changes for gRPC will be a 1.14.x thing. It has not been backported to 1.13.x from what I can see.

https://github.com/quarkusio/quarkus/commit/fd713d371b10320dc8a6f9cb85fb9846255d64ac

Reply:
Well the gRPC tests passed so I guess its fine to have this on master now.

Reply:
Oh, sorry, I thought, this change was mistakenly marked as Quarkus 1.14.0.Final. But yes, let's merge as long as the test is passing